mkdocs.yml

```diff
@@ -40,6 +40,10 @@ nav:
                 - configuration/dashboards.md
           - Supported Data: configuration/data-types.md
     - Development: development/index.md
+    - Testing:
```

Comment:
Lets call the section `CI/CD and Testing`. In there, lets have a page for each of:
* `Testing` (Unit tests, component tests, E2E tests, test coverage) - I would not have separate pages for each of them, as some of them would be rather empty probably
* `Continuous Integration` (Automation, the different GitHub Actions, Dependabot for version upgrades in combination with tests to ensure compatibility )
* `Continuous Deployment` (automatic deployment to Netlify for production and PR previews, automatic deployment of the documentation for production and PR previews)